models/Document/PageSnippet.php

```diff
@@ -99,20 +100,15 @@ abstract class PageSnippet extends Model\Document
     /**
      * @var array
      *
-     * @deprecated since v6.7 and will be removed in Pimcore 10. Use getter/setter methods or $this->inheritedEditables
+     * @deprecated since v6.7 and will be removed in Pimcore 10.
```

Comment:
```suggestion
     * @deprecated since v6.7 and will be removed in Pimcore 10.
     *                From 6.9 on this property will be private instead of protected.
```